docs/SOPs/Managed_access/Data_Transfer_SOP.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Data Transfer SOP
+parent: Managed access
+grand_parent: SOPs
+nav_order: 4
+last_modified_date: 30/05/2024
+---
+
+<script src="https://kit.fontawesome.com/fc66878563.js" crossorigin="anonymous"></script>
+
+# Managed access dataset - Data Transfer SOP
+Once the data contributor has notified the HCA data wrangler that they have completed the  metadata spreadsheet, the wrangler can give them access to hca-util area
```

Comment:
'hca-util area' --> secure storage area

I think hca-util is the tool, let's call the 'area' with one name always.